Feed.py

```diff
@@ -243,7 +223,102 @@ def processRSS():
+    # config.wire_override, \               # Specified as positional arg. #1
+     ])
+    arg_parser.add_argument("wire", type=int, help="The wire no. for feed")
+    arg_parser.add_argument("station", metavar="station-id", type=str,
+                            help="The station identifier for the feed")
+    arg_parser.add_argument("uri",
+            help="The URI for the feed (e.g. http://rss.cnn.com/rss/cnn_topstories.rss or file://civilwar.xml)")
+    arg_parser.add_argument("speed", type=int, help="The code speed for the feed (in WPM)")
+
+    arg_parser.add_argument("--article-pause", "-P", metavar="<sec>", type=float, default=2.0,
+                            help="Pause between articles", dest= "artPause")
+    arg_parser.add_argument("--group-pause", "-G", metavar="<sec>", type=float, default=5.0,
+                            help="Pause between article groups", dest="grpPause")
+    arg_parser.add_argument("--days", "-d", metavar="<days>", type=int, default=0,
+                            help="Number of days from today of articles to read before repeating (default: all)", dest="days")
+    arg_parser.add_argument("--wait", "-w", metavar="<sec>", type=float, default=0.0,
+                            help="Number of seconds to wait for the wire to be idle before sending (default: none)", dest="wait")
+    
+    args = arg_parser.parse_args()
+  # print("arg_parser returned", args)
+    
+    # Wire number for feed:
+    wire = args.wire
+
+    # Station ID for feed:
+    idText = args.station
+
+    # Feed URI:
+    uri = args.uri
+    
+    # The code speed for the feed:
+    wpm = args.speed
+
+    # The cwpm:
+    cwpm = args.min_char_speed
+
+    # Code type: American or International
+    args_code_type = args.code_type.upper()
+    if args_code_type == "A" or args_code_type == "AMERICAN":
+        code_type = config.CodeType.american
+    elif args_code_type == "I" or args_code_type =="INTERNATIONAL":
+        code_type = config.CodeType.international
+    else:
+        msg = "TYPE value '{}' is not a valid `Code Type` value of 'AMERICAN' or 'INTERNATIONAL'.".format(s)
+        log.err(msg)
+        raise ValueError(msg)
+    
+    # Pause between articles (in seconds):
+    artPause = args.artPause
+
+    # Default group pause to article pause value if no group pause value is supplied
+    grpPause = args.grpPause if args.grpPause > 0.0 else args.artPause
+
+    # Number of days (from today) of articles to read before repeating:
+    days = args.days
+
+    # The wait time (in seconds) after someone else transmits before resuming feed:
+    wait = args.wait
+
+    playback_finished = threading.Event()
+    playback_last_sender = None
+
+    mySender = morse.Sender(wpm, cwpm, codeType=code_type)
+    myInternet = internet.Internet(idText)
+    audio_setting = strtobool(str(args.sound))
+    myKOB = kob.KOB(port=config.serial_port, interfaceType=config.interface_type, audio=audio_setting)
```

Comment:
Comment in the ongoing thread.  It's a mistake - that call should be using "args.xxx" (which defaults to the config setting but can be overridden) instead of directly referencing "config.xxx".  Using "-a OFF" should now turn off audio.

Note that I separately filed issue #253 to add an option to Feed.py to feed w/o regard for connections but that's a separate follow-up.